### ISSUE TITLE
Upload the SSSOM mappings done between FOODON and FOODEX2

### ISSUE THREAD

**Issue by jonquet** [agroportal/project-management#533]
This tracker is about uploading in AgroPortal the following mappings: 
https://zenodo.org/records/10716098

They are between FOODON and FOODEX2.
However, me might need to load another version of FOODEX2 before, the one done by @rombaum and @julian-schneider

**Comment by jonquet** [comment]
Also, they are simple mappings to generate also between FOODON and FOODEX2 as there is a branch of FOODON incorporating FOODEX terms : 
<img width="659" alt="Capture d’écran 2024-04-30 à 22 13 55" src="https://github.com/agroportal/project-management/assets/7379183/c5e3b67c-d01a-496b-b64a-4d5e12139b40">

See for instance: https://agroportal.lirmm.fr/ontologies/FOODON?p=classes&conceptid=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FFOODON_03540358

**Comment by imadbourouche** [comment]
@jonquet is this still relevant ?

**Comment by jonquet** [comment]
Yes. This is still relevant. One day we shall have a mapping uploading campain to address all these issues related to the mapping repository.
